src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int BALL_NUMBER = 100;
+
+    public Ball getRandomBall() {
+        Ball newRandomBall = new Ball();
+        ColorSupplier coloursSupplier = new ColorSupplier();
+        Colors randomColor = coloursSupplier.getRandomColor();
+        newRandomBall.setColor(randomColor.name());
+        Random random = new Random();
```

Comment:
make Random and ColorSupplier global fields like you did in a ColorSupplier with Random